scripts/remove/lounge.sh

```diff
@@ -3,10 +3,12 @@
 systemctl disable -q lounge >> /dev/null 2>&1
 systemctl stop -q lounge
 
-npm uninstall -g thelounge --save >> /dev/null 2>&1
```

Comment:
Wonder if we should just keep this? In case someone never runs upgrade but decided to remove the lounge? 🤔

Reply:
Yeah, that seems smort. Will add it back.